cfgov/unprocessed/css/forms.less

```diff
@@ -650,21 +441,315 @@
     }
 });
 
+
+/* topdoc
+  name: Checkboxes and radio buttons
+    family: cf-forms
+    notes:
+      - "CSS for checkboxes and radio buttons requires following a specific HTML structure for each type of input, otherwise the styles will not be properly applied."
+      - "You MUST wrap your input.cf-input in a label."
```

Comment:
Thanks for the catch. I'll go over the notes, I copied and pasted from the old custom code.